freqtrade/data/btanalysis.py

```diff
@@ -121,9 +141,54 @@ def load_backtest_stats(filename: Union[Path, str]) -> Dict[str, Any]:
     with filename.open() as file:
         data = json_load(file)
 
+    data['metadata'] = load_backtest_metadata(filename)
     return data
 
 
+def find_existing_backtest_stats(dirname: Union[Path, str], run_ids: Dict[str, str]) -> Dict[str, Any]:
+    """
+    Find existing backtest stats that match specified run IDs and load them.
+    :param dirname: pathlib.Path object, or string pointing to the file.
+    :param run_ids: {strategy_name: id_string} dictionary.
+    :return: results dict.
+    """
+    # Copy so we can modify this dict without affecting parent scope.
+    run_ids = copy(run_ids)
+    dirname = Path(dirname)
+    results = OrderedDict([
+        ('metadata', {}),
+        ('strategy', {}),
+        ('strategy_comparison', []),
+    ])
+
+    for filename in reversed(sorted(dirname.iterdir())):
+        if (not filename.name.startswith('backtest-result') or
+            not filename.name.endswith('.json') or
+            filename.name.endswith('.meta.json')):
+            continue
```

Comment:
I think you should be able to use `glob()` here - something like `dirname.glob('backtest-resut.*meta.json)` ... which would simplify this whole segment to just one line